common/src/main/java/dev/lavalink/youtube/track/YoutubeAudioTrack.java

```diff
@@ -66,6 +70,14 @@ public void process(LocalAudioTrackExecutor localExecutor) throws Exception {
     }
 
     try (HttpInterface httpInterface = sourceManager.getInterface()) {
+      try {
+        JsonObject userData = JsonParser.object().from(getUserData().toString());
```

Comment:
This ideally should use Lavaplayer's own `JsonBrowser` class. Deprecating use of nanojson is on my todo list as it doesn't make sense to be using multiple JSON libraries here, and is largely an oversight on my part.